tensorflow/lite/micro/micro_allocation_info.h

```diff
@@ -63,7 +63,8 @@ struct GraphAllocationInfo {
 // `Finish`.
 class AllocationInfoBuilder {
  public:
-  AllocationInfoBuilder(const Model* model, SimpleMemoryAllocator* allocator,
+  AllocationInfoBuilder(const Model* model,
+                        INonPersistentBufferAllocator* allocator,
```

Comment:
Is it worthwhile calling out the "allocator" to be non-persistent? Like calling it "non_persistent_allocator", or is there a specific reason why we shouldn't do this?

Reply:
Good call! I agree. Changed